src/main/java/slimeknights/tconstruct/tables/client/inventory/module/SideButtonsScreen.java

```diff
@@ -30,11 +30,13 @@ public void addSideButton(Button button) {
     int rows = (this.buttonCount - 1) / this.columns + 1;
 
     this.xSize = button.getWidth() * this.columns + this.spacing * (this.columns - 1);
-    this.ySize = button.getHeight() * rows + this.spacing * (rows - 1);
+    // TODO: getWidth_CLASH->getHeight()
+    this.ySize = button.getWidth_CLASH() * rows + this.spacing * (rows - 1);
```

Comment:
`getHeight` still exists. For some dumb reason Mojang has two getters for height, one of which had a mapping conflict.